### PR TITLE
It should not just be HTTP

### PR DESCRIPTION
**Pull request by matzew**
state that HTTP is just initial focus

### CODE REVIEW

File: docs/spec/motivation.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ decoupled way.
 
 Kubernetes has no primitives related to event processing, yet this is an
 essential component in serverless workloads. Eventing introduces high-level
-primitives for event production and delivery with a focus on push over HTTP. If
+primitives for event production and delivery with an _initial_ focus on push over HTTP. If
```
Comment (n3wscott) [inline docs/spec/motivation.md]: Looks good, but needs the wrapping fixed.

**Comment by matzew** [comment]
@n3wscott done!

**Comment by n3wscott** [comment]
> done!

maybe not pushed?

**Comment by matzew** [comment]
@n3wscott better ?

**Pull request merged**